python/si-validator.py

```diff
@@ -74,6 +83,27 @@ def save_yaml(yaml_dict, path):
         file.write(yaml_dict)
 
 
+def get_schema_from_gh(url):
+    response = requests.get(url)
+    response.raise_for_status()
+    return response.text
+
+
+def schema_yaml_from_source(schema_source):
+    if schema_source == "latest":
+        schema = get_schema_from_gh(
+            "https://raw.githubusercontent.com/ossf/security-insights-spec/main/security-insights-schema"
```

Comment:
This seems more like "edge" than latest, right? If we can grab the latest release version (didn't look very far into that yet) then we could reach out to grab that version tag instead of pinning to the edge source file

Reply:
@eddie-knight done :) good proposal!

Reply:
Great solution!